### PR TITLE
Feature/provide switch to enable native build

### PR DESCRIPTION
**Pull request by ap891843**
provide switch to enable native build

### CODE REVIEW

File: clients/cobol-lsp-vscode-extension/package.json [hunk 1]
```diff
@@ -438,6 +438,11 @@
                         "S999"
                     ],
                     "uniqueItems": true
+                },
+                "cobol-lsp.enableNativeBuild": {
```
Comment (ishche) [inline clients/cobol-lsp-vscode-extension/package.json:442]: The configuration parameter name looks confusing to me. Will enableNativeServer be a better name? Maybe @zeibura or @zimlu02 can add something here?

**Review by ishche — CHANGES_REQUESTED** [review]

**Review by niharikasw91 — APPROVED** [review]
Looks good!!!

File: clients/cobol-lsp-vscode-extension/package.json (hunk 1 above)
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/package.json:442]: Server type: Java / Native

File: clients/cobol-lsp-vscode-extension/package.json [hunk 2]
```diff
@@ -442,6 +442,16 @@
                         "S999"
                     ],
                     "uniqueItems": true
+                },
+                "cobol-lsp.serverType": {
+                    "type": "string",
+                    "enum": ["JAVA", "NATIVE"],
```
Comment (ap891843) [inline clients/cobol-lsp-vscode-extension/package.json:448]: @zeibura @zimlu02 @ishche  Updated

**Review by ishche — APPROVED** [review]

File: clients/cobol-lsp-vscode-extension/package.json (hunk 2 above)
Comment (ap891843) [reply clients/cobol-lsp-vscode-extension/package.json:448]: image

Comment (ap891843) [reply clients/cobol-lsp-vscode-extension/package.json:448]: Description updated to - COBOL LS server type

**Pull request merged**